code/modules/admin/verbs/striketeam.dm

```diff
@@ -1,31 +1,41 @@
-//STRIKE TEAMS
+//Deathsquad
 
 #define COMMANDOS_POSSIBLE 6 //if more Commandos are needed in the future
-GLOBAL_VAR_INIT(sent_strike_team, 0)
+GLOBAL_VAR_INIT(sent_strike_team, FALSE)
 
 /client/proc/strike_team()
-	if(!SSticker)
-		to_chat(usr, "<span class='userdanger'>The game hasn't started yet!</span>")
+	if(!check_rights(R_EVENT))
 		return
-	if(GLOB.sent_strike_team == 1)
-		to_chat(usr, "<span class='userdanger'>CentComm is already sending a team.</span>")
+	if(SSticker.current_state == GAME_STATE_PREGAME)
+		to_chat(usr, "<span class='warning'>The round hasn't started yet!</span>")
 		return
-	if(alert("Do you want to send in the CentComm death squad? Once enabled, this is irreversible.",,"Yes","No")!="Yes")
+	if(GLOB.sent_strike_team == TRUE)
+		if(alert("A Deathsquad is already being sent, are you sure you want to send another?",, "Yes", "No") != "Yes")
+			return
+	message_admins("<span class='notice'>[key_name_admin(usr)] has started to spawn a DeathSquad.</span>")
+	log_admin("[key_name_admin(usr)] has started to spawn a DeathSquad.")
+	if(alert("Do you want to send in the Deathsquad? Once enabled, this is irreversible.",, "Yes", "No") != "Yes")
 		return
 	alert("This 'mode' will go on until everyone is dead or the station is destroyed. You may also admin-call the evac shuttle when appropriate. Spawned commandos have internals cameras which are viewable through a monitor inside the Spec. Ops. Office. The first one selected/spawned will be the team leader.")
 
-	message_admins("<span class='notice'>[key_name_admin(usr)] has started to spawn a CentComm DeathSquad.</span>", 1)
-
 	var/input = null
 	while(!input)
-		input = sanitize(copytext(input(src, "Please specify which mission the death commando squad shall undertake.", "Specify Mission", ""),1,MAX_MESSAGE_LEN))
+		input = sanitize(copytext(input(src, "Please specify which mission the Deathsquad shall undertake.", "Specify Mission", "",), 1, MAX_MESSAGE_LEN))
 		if(!input)
```

Comment:
this if statement will always return FALSE, perhaps checking if input is empty instead of null will be better here. With the current code it's impossible to cancel the deathsquad.